src/app/main/connection-window/connection-window.component.scss

```diff
@@ -160,7 +164,9 @@ $secondary-color-green: #b7e2ad;
       bottom: 10px;
       left: 50%;
       transform: translateX(-50%);
-      a, a:hover {
+
+      a::before,
+      a:hover::before {
```

Comment:
I dont know if this is correct, that is up to you experts to decide.
But it did fix codacy :D

Reply:
shouldn't it be:

```
a,
a:hover {
```
I don't see why we should add `::before`

Reply:
a,
a:hover 

makes Codacy mad

I read some and ::before fixed the issue, but as I said above I don't know if it is the correct way of doing it

Reply:
ok I need to have a look at that, everything else is good